modules/Emails/Email.php

```diff
@@ -4684,15 +4711,19 @@ private function sendOptInEmailToEmailAddressById($id)
             isset($sugar_config['email_enable_auto_send_opt_in'])
             && $sugar_config['email_enable_auto_send_opt_in']
         ) {
-            /** @var \EmailAddress $emailAddress */
+            /** @var EmailAddress $emailAddress */
             $emailAddresses = BeanFactory::getBean('EmailAddresses');
             $emailAddress = $emailAddresses->retrieve($id);
 
             if (
                 $emailAddress !== null
                 && $emailAddress->getConfirmedOptInState() != EmailAddress::COI_STAT_CONFIRMED_OPT_IN
-                && empty($emailAddress->confirm_opt_in_sent_date)) {
-                $this->sendOptInEmail($emailAddress);
+                && empty($emailAddress->confirm_opt_in_sent_date)
+            ) {
+                $ret = $this->sendOptInEmail($emailAddress);
```

Comment:
This is a deprecated non-working function, you won't be able to send opt-in emails.

Reply:
true. but the function call already been there.

Reply:
Just noting we need to fix this or opt-in is broken. Can do it in another PR though.

Reply:
Opt-in functionality is working correctly now, another PR must have already addressed this.